fixed timezone diff (#37)
badgerloop-software/chase-car-dashboard: README.md

1. If you are not familiar with Docker, it is recommended to install Docker Desktop.
2. Once it is installed, start Docker Desktop. If Docker is not started, you will see a message saying Docker Daemon is Starting. Wait for this to complete.
3. Look for a script in this GitHub repo that matches your OS. If you're on Windows, that script is called `windows-pull-run.bat`, and for Linux and Mac it is called `linux-pull-run.sh`.

> <<<<<<< HEAD
> 4. To download the script, open the script in GitHub and then click "Raw". The file will open in a tab. Then do `Ctrl + S` or `Command + S`, depending on your OS, and save it. This script will create a folder called recordedData in the same spot as your script, so keep that in mind when choosing where to download it.  If operating on Windows make sure to remove the ".txt" when downloading. To run, look at the bullet points below:
>    - (Windows) Navigate to the folder storing the script and double click on it. 
>    - (MacOS/Linux) `cd` into the directory storing the script, and then run these commands: `chmod +x linux-pull-run.sh` and `./linux-pull-run.sh`.
> =======
> 4. To download the script, open the script in GitHub and then click "Raw". The file will open in a tab. Then do `Ctrl + S` or `Command + S`, depending on your OS, and save it. This script will create a folder called recordedData in the same spot as your script, so keep that in mind when choosing where to download it. If operating on Windows make sure to remove the ".txt" when downloading and also download the powershell script in same folder. To run, look at the bullet points below:
>    - (Windows) Navigate to the folder storing the script and double click on it. 
>    - (MacOS/Linux) `cd` into the directory storing the script, and then run these commands: `chmod +x linux-pull-run.sh`, and `./linux-pull-run.sh`.
> >>>>>>> 10dd8a5 (after updating my pull testing-changes)

Reviewer: ```suggestion
4. To download the script, open the script in GitHub and then click "Raw". The file will open in a tab. Then do `Ctrl + S` or `Command + S`, depending on your OS, and save it. This script will create a folder called recordedData in the same spot as your script, so keep that in mind when choosing where to download it. If operating on Windows make sure to remove the ".txt" when downloading and also download the powershell script in same folder. To run, look at the bullet points below:
   - (Windows) Navigate to the folder storing the script and double click on it. 
   - (MacOS/Linux) `cd` into the directory storing the script, and then run these commands: `chmod +x linux-pull-run.sh` and `./linux-pull-run.sh`.
```

1. If you are not familiar with Docker, it is recommended to install Docker Desktop.
2. Once it is installed, start Docker Desktop. If Docker is not started, you will see a message saying Docker Daemon is Starting. Wait for this to complete.
3. Look for a script in this GitHub repo that matches your OS. If you're on Windows, that script is called `windows-pull-run.bat`, and for Linux and Mac it is called `linux-pull-run.sh`.

> 4. To download the script, open the script in GitHub and then click "Raw". The file will open in a tab. Then do `Ctrl + S` or `Command + S`, depending on your OS, and save it to your Downloads folder. If operating on Windows make sure to remove the ".txt" when downloading and also download the powershell script in same folder To run, look at the bullet points below:
>    - (Windows) Navigate to the Downloads folder and double click on the script. 
>    - (MacOS/Linux) Run these commands in your terminal `cd Downloads`, `chmod +x linux-pull-run.sh`, and `./linux-pull-run.sh`.

Reviewer: ```suggestion
4. To download the script, open the script in GitHub and then click "Raw". The file will open in a tab. Then do `Ctrl + S` or `Command + S`, depending on your OS, and save it. This script will create a folder called recordedData in the same spot as your script, so keep that in mind when choosing where to download it. If operating on Windows make sure to remove the ".txt" when downloading and also download the powershell script in same folder. To run, look at the bullet points below:
   - (Windows) Navigate to the folder storing the script and double click on it. 
   - (MacOS/Linux) `cd` into the directory storing the script, and then run these commands: `chmod +x linux-pull-run.sh`, and `./linux-pull-run.sh`.
```

>    - (MacOS/Linux) `cd` into the directory storing the script, and then run these commands: `chmod +x linux-pull-run.sh`, and `./linux-pull-run.sh`.

Reviewer: ```suggestion
   - (MacOS/Linux) `cd` into the directory storing the script, and then run these commands: `chmod +x linux-pull-run.sh` and `./linux-pull-run.sh`.
```
